connect/runtime/src/test/java/org/apache/kafka/connect/integration/SessionedProtocolIntegrationTest.java

```diff
@@ -162,7 +162,7 @@ public void ensureInternalEndpointIsSecured() throws Throwable {
         );
         assertEquals(
             FORBIDDEN.getStatusCode(),
-            connect.executePost(connectorTasksEndpoint, "[]", invalidSignatureHeaders)
+            connect.executePost(connectorTasksEndpoint, "[]", invalidSignatureHeaders).getStatus()
```

Comment:
Technically, we shouldn't *have* to change this test if we were to maintain backward compatibility as I suggested on a different comment. But if we were to deprecate the existing `executePost` method that returns an int (and similar methods), I think it's worth changing this test to use the new methods that do return the `Response` to prevent a deprecation warning and to show users an example of how to migrate to the new methods.

Reply:
I agree. Using the new methods now.